### PR TITLE
add spinner when user clicks "Download Data"

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
also removed query-builder styling and translations

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts [hunk 1]
```diff
@@ -115,9 +119,13 @@ export class AggregateReportComponent implements OnInit, OnDestroy {
...
 
   onDownloadDataButtonClick(): void {
+    this.spinnerModal.loading = true;
     this.reportService.getReportContent(this.report.id)
       .subscribe(download => {
+        this.spinnerModal.loading = false;
         saveAs(download.content, download.name);
+      }, error => {
+        this.spinnerModal.loading = false;
       });
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts]: tip: there is a third callback to subscribe method called "completed" which is called after success and error that you could use to put the loading = false in

**Comment by esotrope** [comment]
may want this on the "My Reports" page as well now that i think about it

**Comment by dsnyder-fairway** [comment]
@esotrope good call. I'll add this to downloads there

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts (hunk 1 above)
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts]: I wasn't able to add this for completed (specifically, on failures). Seeing the api docs shows for the completed param:
```
A handler for a terminal event resulting from successful completion.
```

**Comment by dsnyder-fairway** [comment]
@esotrope please review my latest changes for the My Reports page

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts (hunk 1 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts]: Looks like you're right : /
I guess there is an RxJS "finally" method that can be used for this functionality - kind of strange

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.ts]: 👍, added!

**Pull request merged**